students/leejiyub/users/views.py

```diff
@@ -33,4 +32,19 @@ def post(self, request):
 
             return JsonResponse({'message' : 'SUCCESS'}, status=201)
         except KeyError:
-            return JsonResponse({'message' : 'KeyError'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message' : 'KeyError'}, status=400)
+
+class LoginView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+       
+        try:
+            email = data['email']
+            password = data['password']
+
+            if not User.objects.filter(email = data["email"], password = data["password"]).exists():
+                return JsonResponse({'message' : 'INVALID_USER'}, status=401)
```

Comment:
위에서 변수화를 했다면 활요해 주세요.

```suggestion
            if not User.objects.filter(email = email, password = password).exists():
                return JsonResponse({'message' : 'INVALID_USER'}, status=401)
```

Reply:
넵 수정했습니다